Add WasmEdge integration docs (#327)
OpenFunction/openfunction.dev: content/en/docs/getting-started/Quickstarts/prerequisites.md

   ```shell
   $ kafkacat -L -b <kafka-server>-kafka-brokers:9092
   ```

## WasmEdge

> Function now supports using `WasmEdge` as workload runtime, here you can find steps to setup `WasmEdge` workload runtime in kubernetes cluster.

Reviewer: ```suggestion
Function now supports using `WasmEdge` as workload runtime, here you can find steps to setup the `WasmEdge` workload runtime in a Kubernetes cluster (with `containerd` as the container runtime).
```

> > The following steps should be executed on all your nodes that want to run the function with WasmEdge workload runtime

Reviewer: ```suggestion
> You should run the following steps on all the nodes (or a subset of the nodes that will host the wasm workload) of your cluster.
```

1. **WasmEdge**

>    The easiest way to install WasmEdge is to run the following command. Your system should have git and curl as prerequisites.

Reviewer: ```suggestion
   The easiest way to install WasmEdge is to run the following command. Your system should have git and curl installed.
```
